src/postprocess/band_structure.jl

```diff
@@ -30,3 +30,20 @@ function compute_bands(ham::Hamiltonian, kpoints, n_bands;
 
     select_eigenpairs_all_kblocks(band_data, 1:n_bands)
 end
+
+function plot_bands(ham, n_bands, kline_density, composition, εF)
+    basis = ham.basis
+    # Band structure calculation along high-symmetry path
+    kpoints, klabels, kpath = high_symmetry_kpath(basis, kline_density, composition...)
+    println("Computing bands along kpath:\n     $(join(kpath[1], " -> "))")
+    band_data = compute_bands(ham, kpoints, n_bands)
+
+    # Plot bandstructure using pymatgen
+    plotter = pyimport("pymatgen.electronic_structure.plotter")
+    bs = pymatgen_bandstructure(basis, band_data, klabels, fermi_level=εF)
+    bsplot = plotter.BSPlotter(bs)
+    plt = bsplot.get_plot()
+    plt.autoscale()
+    plt.legend()
+    plt.show()
```

Comment:
Having a call to `show()` in a library function is a bad idea in my opinion, because a user might want to add more stuff to the plot, which you prevent in this way. Also actually the legend thing does not work properly anyway ... so you can remove that, too ;).

Reply:
I don't really see that as a library function, postprocessing is like a collection of utility functions you can call to help your analyses. If it doesn't do exactly what you want you take the code out and tweak it. I removed the legend.

Reply:
Also the show works differently in Julia and can actually be removed without any impact (I just tried it). Still, returning the `plt` object to the user is probably helpful.